app/javascript/react/screens/App/Overview/screens/PlanWizard/components/PlanWizardResultsStep/PlanWizardResultsStepActions.js

```diff
@@ -35,3 +39,21 @@ const _postMigrationPlansActionCreator = (url, migrationPlans, planSchedule) =>
 
 export const postMigrationPlansAction = (url, migrationPlans, planSchedule) =>
   _postMigrationPlansActionCreator(url, migrationPlans, planSchedule);
+
+const _putMigrationPlansActionCreator = (url, planId, migrationPlans, planSchedule) => dispatch =>
+  dispatch({
+    type: PUT_V2V_MIGRATION_PLANS,
+    payload: new Promise((resolve, reject) => {
+      API.put(`${url}/${planId}`, migrationPlans)
+        .then(response => {
+          resolve(response);
+          if (planSchedule === 'migration_plan_now') {
+            postMigrationRequestsAction(response, dispatch);
+          }
+        })
+        .catch(e => reject(e));
+    })
+  });
+
+export const putMigrationPlansAction = (url, migrationPlans, planSchedule) =>
```

Comment:
There is something going on with this function signature.  Should there be four arguments with the second being `id`?

Reply:
Wow, how did that even work... I guess it was dropping the actual `planSchedule` argument, thanks for that catch!